doc/examples/anim04/anim04.sh

```diff
@@ -17,15 +17,16 @@ cat << EOF > pre.sh
 # Set up flight path
 gmt begin
 	gmt project -C-73.8333/40.75 -E-80.133/25.75 -G10 -Q > flight_path.txt
-	gmt grdgradient @USEast_Coast.nc -A90 -Nt1 -Gint_US.nc
+	gmt grdcut @earth_relief_02m -R-100/-70/10/50 -GUSEast_Coast.nc
+	gmt grdgradient USEast_Coast.nc -A90 -Nt1 -Gint_US.nc
 	gmt makecpt -Cglobe -H > globe_US.cpt
 gmt end
 EOF
 # 2. Set up the main frame script
 cat << EOF > main.sh
 gmt begin
 	gmt grdimage -JG\${MOVIE_COL0}/\${MOVIE_COL1}/160/210/55/0/36/34/\${MOVIE_WIDTH}+ \
```

Comment:
Note the trailing `+` in the -J option. Is it an obsolete syntax?

Reply:
Yes it is.  Means the max dimension so +du in new syntax.

Reply:
I've changed it to the new syntax.